### ISSUE TITLE
Lint error with httpapi variables var-naming[no-role-prefix]

### ISSUE THREAD

**Issue by RomainTiennot** [ansible-collections/community.zabbix#1011]
##### SUMMARY
Since updating ansible-lint to version 6.16.1, they have updated the var-naming rule. I get several errors because of the following variables: 
```
    ansible_network_os:
    ansible_user:
    ansible_httpapi_pass:
    ansible_httpapi_port:
    ansible_httpapi_use_ssl:
    ansible_httpapi_validate_certs:
    ansible_zabbix_url_path:
```
Example:
`var-naming[no-role-prefix]: Variables names from within roles should use role_name_ as a prefix. (set_fact: ansible_httpapi_pass)`

I've updated ansible-lint and I still have the problem.
The rule seems rather coherent and would require an update of the variables in the community.zabbix project.

COMPONENT NAME
community.zabbix.zabbix httpapi

ANSIBLE VERSION
ansible [core 2.15.0]
  config file = /home/romain/project/ansible/ansible.cfg
  configured module search path = ['/home/romain/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/romain/.local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/romain/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/romain/.local/bin/ansible
  python version = 3.10.0 (default, Feb 19 2023, 19:26:21) [GCC 10.2.1 20210110] (/usr/local/bin/python3.10)
  jinja version = 3.1.2
  libyaml = True

OS / ENVIRONMENT / Zabbix Version
OS: Debian 11.6
Zabbix: 6.2.7
Zabbix Collection: 2.0.1
Zabbix API: 0.5.6

**Comment by pyrodie18** [comment]
With the exception of `ansible_zabbix_url_path` these are all ansible vars that the role happens to use.  Not sure that there's a lot we can do about that.

**Comment by RomainTiennot** [comment]
Hello,
Thank you for your reply.
We can close the issue. This has been fixed with ansible-lint version 6.17.1.
Have a nice day!
